stores/views.py

```diff
@@ -16,46 +17,64 @@
 AfterShaveSkinType,
 SkinType,
 Color,
-Cart
+Cart,
+Order
 )
 
 class CartAddView(View):
+    @login_required
     def post(self, request):
         data = json.loads(request.body)
         try:
+            if not Order.objects.filter(user = request.user, order_status_id = 1).exists():
```

Comment:
클래스에 enum로 선언해서 만약 1이 결제대기를 나타낸다면 ORDER_STATUS_PENDING = 1 다음과 같이 사용하면 가독성 조금 좋아지지 않을까요?
```suggestion
            if not Order.objects.filter(user = request.user, order_status_id = ORDER_STATUS_PENDING).exists():
```